io/features/eu.esdihumboldt.hale.io.feature.core/feature.xml

```diff
@@ -280,4 +280,11 @@
          version="0.0.0"
          unpack="false"/>
 
+   <plugin
```

Comment:
I'm missing an additional entry for the xtraserver-config-util bundle here, i.e.

```xml
   <plugin
         id="de.interactive_instruments.xtraserver-config-util"
         download-size="0"
         install-size="0"
         version="0.0.0"
         unpack="false"/>
```